tests/sentry/api/endpoints/test_team_stats.py

```diff
@@ -16,18 +16,22 @@ def test_simple(self):
         team_2 = self.create_team(members=[self.user])
         project_3 = self.create_project(teams=[team_2], name="c")
 
-        tsdb.incr(tsdb.models.project, project_1.id, count=3)
-        tsdb.incr(tsdb.models.project, project_2.id, count=5)
-        tsdb.incr(tsdb.models.project, project_3.id, count=10)
+        for project, count in ((project_1, 2), (project_2, 1), (project_3, 4)):
+            for _ in range(count):
+                self.store_event(
+                    data={
+                        "timestamp": iso_format(before_now(minutes=5)),
+                    },
+                    project_id=project.id,
+                )
 
         url = reverse(
             "sentry-api-0-team-stats",
             kwargs={"organization_slug": team.organization.slug, "team_slug": team.slug},
         )
         response = self.client.get(url)
-
         assert response.status_code == 200, response.content
-        assert response.data[-1][1] == 8, response.data
+        assert response.data[-1][1] == 3, response.data
```

Comment:
Hmm, this one seems a bit sus. Was there a flaw in the original code that resulted in 8?

Reply:
I just changed the amount that we're storing. We used to store 3 for project 1, and 5 for project 2. To store things now, we need to store an event to make up the count. To save on unnecessary IO i just changed how many events we store for each project, which changes the final counts here

Reply:
Ahh, I was being dumb and read it wrong. Your explanation makes sense to me.